search_bytes/searching_aes_keys.md

```diff
@@ -1,8 +1,27 @@
 ## Searching for AES Keys
 
-Thanks to Victor Muñoz, radare2 now has support of the algorithm he developed, capable of finding expanded AES keys with `/Ca` command. It searches from current seek position up to the `search.distance` limit, or until end of file is reached. You can interrupt current search by pressing `Ctrl-C`. For example, to look for AES keys in physical memory of your system:
+Thanks to Victor Muñoz, radare2 now has support of the algorithm he developed, capable of finding **expanded AES** keys with `/Ca` command. It searches from current seek position up to the `search.distance` limit, or until end of file is reached. You can interrupt current search by pressing `Ctrl-C`. For example, to look for AES keys in physical memory of your system:
+
 ```
 $ sudo r2 /dev/mem
 [0x00000000]> /Ca
 0 AES keys found
 ```
+
+If you are simply looking for plaintext AES keys in your binary, `/Ca` will not find them, but you might want to search with `is~AES` instead if the programmer left those hints for you:
```

Comment:
aes stuff can be also referenced from strings , not just symbols. also if the program is using an external implementation it will be available in the imports

Reply:
Right, good point. Then the other commands that you would recommend would be `/` and/or `pd~` or similar for this case? Is there any specific command to search in the imports?